board/drivers/bxcan.h

```diff
@@ -181,6 +182,11 @@ void can_rx(uint8_t can_number) {
   CAN_TypeDef *CAN = CANIF_FROM_CAN_NUM(can_number);
   uint8_t bus_number = BUS_NUM_FROM_CAN_NUM(can_number);
 
+  // Enable automatic retransmission of messages after some activity on the bus
+  if (can_health[can_number].total_rx_cnt != 0U && (CAN->MCR & CAN_MCR_NART) != 0U) {
+    CAN->MCR &= ~(CAN_MCR_NART);
```

Comment:
let's just do this for the H7 for now

Reply:
As discussed - auto retry was always enabled on F4